bemlint/lib/rules/element.js

```diff
@@ -0,0 +1,64 @@
+'use strict';
+
+const createError = require('../util/createError');
+const getParentBlock = require('../util/getParentBlock');
+const { element: massages } = require('./errorMassage.json')
+
+module.exports = [
```

Comment:
I mean make Object.values just once instead of doing it in the loop, it is great to have an object with some rule id as key

Reply:
![image](https://user-images.githubusercontent.com/60401137/108822534-e2444a00-75c7-11eb-83b4-5ab735e388a5.png)